actions/icinga.remove.downtime.yaml

```diff
@@ -0,0 +1,29 @@
+---
+description: Remove a downtime for Host or Service in Icinga
+enabled: true
+entry_point: src/openstack_actions.py
+name: icinga.remove.downtime
+parameters:
+  name:
+    type: string
+    description: Name of host or service in Icinga
+    required: true
+  object_type:
+    immutable: false
+    type: string
+    description: Icinga Object Type
```

Comment:
maybe give some examples - or an enum would be handy here

Reply:
object_type is an enum, either Host or Service

Reply:
ignore - I didn't see its already an enum :)